front50-core/src/test/groovy/com/netflix/spinnaker/front50/model/plugins/PluginInfoServiceSpec.groovy

```diff
@@ -70,9 +72,12 @@ class PluginInfoServiceSpec extends Specification {
 
     then:
     1 * validator.validate(newPluginInfo, _)
-    1 * repository.findById("foo.bar") >> currentPluginInfo
+    2 * repository.findById("foo.bar") >> currentPluginInfo
     1 * repository.update("foo.bar", newPluginInfo)
     0 * repository.create(_, _)
+    1 * echoService.postEvent(_) >> { args ->
```

Comment:
Is this reasonable to tack on here, or should this be separated out to a new test?